core/src/com/group66/game/screens/ShopScreen.java

```diff
@@ -149,28 +154,50 @@ private void createScreen() {
         // revert the checked state.
         levelButton.addListener(new ChangeListener() {
             public void changed(ChangeEvent event, Actor actor) {
-                BustaMove.getGameInstance().setScreen(new MainMenuScreen());
+                dispose();
+                try {
```

Comment:
If we don't do special stuff for different exceptions, we might as well do
`catch (Exception e)`